js-envs-test-kit/src/main/scala/org/scalajs/jsenv/test/JSEnvSuiteConfig.scala

```diff
@@ -26,42 +26,43 @@ import scala.concurrent.duration._
  */
 final class JSEnvSuiteConfig private (
     val jsEnv: JSEnv,
-    val terminateVMJSCode: Option[String],
+    val supportsExit: Boolean,
     val supportsCom: Boolean,
     val supportsTimeout: Boolean,
     val awaitTimeout: FiniteDuration,
     val description: String
 ) {
   private def this(jsEnv: JSEnv) = this(
       jsEnv = jsEnv,
-      terminateVMJSCode = None,
+      supportsExit = true,
       supportsCom = true,
       supportsTimeout = true,
       awaitTimeout = 1.minute,
       description = jsEnv.name
   )
 
-  def withTerminateVMJSCode(code: String): JSEnvSuiteConfig =
-    copy(terminateVMJSCode = Some(code))
-
   def withSupportsCom(supportsCom: Boolean): JSEnvSuiteConfig =
     copy(supportsCom = supportsCom)
 
   def withSupportsTimeout(supportsTimeout: Boolean): JSEnvSuiteConfig =
     copy(supportsTimeout = supportsTimeout)
 
+  def withSupportsExit(supportsExit: Boolean): JSEnvSuiteConfig =
+    copy(supportsExit = supportsExit)
```

Comment:
Any reason not to put that method where `withTerminateVMJSCode` was? That would keep the ordering so that all fields are always in the same order.

Reply:
Nope, fluke. Good catch.